### ISSUE TITLE
Learning from Long-Tailed Data with Noisy Labels

### ISSUE THREAD

**Issue by SerialLain3170** [SerialLain3170/nagato#60]
## Overview
- Authors leverage two-stage training pipeline in order to deal with long-tailed class distribution and noisy labels
  - In the first stage, authors utilize self-supervised learning for long-tailed class distribution
  - In the finetuning stage, authors use Logit Adjustment loss and SuperLoss for noisy labels

![Screen Shot 2021-10-10 at 9 00 33](https://user-images.githubusercontent.com/32360147/136676711-5a63e2d2-aabc-4c5a-b0fb-16f3fc5c4d86.png)

**Comment by SerialLain3170** [comment]
## Supplemental Information

[Paper Link](https://arxiv.org/pdf/2108.11096.pdf)

### Authors
Shyamgopal Karthik, Jérome Revaud, Chidlovskii Boris

### Organization
IIIT Hyderabad
Naver Labs Europe

### Conference
arXiv preprint 2021
